cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/AppFabricTestBase.java

```diff
@@ -193,6 +197,10 @@ public static void afterClass() throws Exception {
     txManager.stopAndWait();
   }
 
+  protected UsageHandler getUsageHandler() throws IOException, DatasetManagementException {
```

Comment:
This is b/c getting the same `UsageDataset` from `UsageHandler` and from `UsageHandlerTest` results in two different instances of the `UsageDataset`, so they aren't really interacting with the same dataset. Should probably file a bug for this.